vscode/package.json

```diff
@@ -225,6 +225,7 @@
     },
     "dependencies": {
         "cross-zip": "^3.0.0",
+        "subtitles-parser-vtt": "git+https://github.com/rbrisita/subtitles-parser.git",
```

Comment:
While installing from Github works in the short term, it is a very risky business to go into as it has no clear versioning. A dependency can change at any notice with no control on our side. 

Also, I see this was a fork. Why was a fork needed?

Reply:
The repo is controlled by me and was needed to fix up documentation and expose internal functions that convert SRT format to milliseconds and vice versa. The originating repo hasn't had any activity for awhile.

I could create a tag with the version number and use that as the Git URL. For example:
`git+https://github.com/rbrisita/subtitles-parser.git#0.0.9`

Reply:
As long as it is in git, it is unsafe by definition (I realize this is your repo, but it is an anti-pattern. Github is not meant to be used as a package registry as it has no built-in semver safety measures). Either publish an npm package from your fork, or copy the needed function exposed to the repo.

Reply:
There seems to be a misunderstanding here.
* NPM (owned by GitHub) uses Git to get its job done.
* There is semantic versioning with NPM and Git, I'll paste the URL again from above for your convenience: https://docs.npmjs.com/cli/v6/configuring-npm/package-json#git-urls-as-dependencies
* NPM itself is unsafe as there is no vetting process for submissions.

The simplest example to install with semantic versioning:
`npm install "rbrisita/subtitles-parser#semver:~0.0.7"`
Which will install v0.0.9.

I ask that you reconsider the two options you gave as they increase the management of code inside the Codio project.